### ISSUE TITLE
HTML syntax highlighting without "template:" in front of it

### ISSUE THREAD

**Issue by richardkazuomiller** [gandm/language-babel#252]
Thanks very much for this package!

I use template strings for rendering HTML, and in order to get the syntax highlighting I create an object and set the value of of `template`, but this is unnecessary for me, since I'm not using a framework that uses that structure.

Would it be possible to implement something like this?

````
const html = 
/** @html */
`<div>
  <span>Hello world</span>
</div>`

````

It looks like [atom-typescript](https://github.com/richardkazuomiller/atom-typescript/blob/master/grammars/ts.cson#L614) has something like this but I don't understand how to define grammars well enough to implement it myself (^^ゞ

**Comment by gandm** [comment]
Yes. Because this is a more frequent request I'm developing an option via the package settings to extend the grammar. Not sure when I'll be able to finish it - hopefully soon. In essence you will be able to specify the template tag and the scope for your chosen languages 
.e.g. to support two forms of template handling html and sql the config would be.

`/* @html */:text.html.basic, sql:source.sql`

**Comment by gandm** [comment]
@richardkazuomiller I've just published 2.42.0 that adds support for extending the grammar. Please see [the README section here](https://github.com/gandm/language-babel#javascript-tagged-template-literal-grammar-extensions) for more information.

Please not that you must use the /**@ html */ immediately prior to the backtick. No intervening spaces nor new lines.

**Comment by richardkazuomiller** [comment]
Works great. Thanks!
